app/src/main/res/values-ar/about.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:locale="en">
+    <string name="about_close">أغلق</string>
+    <string name="about_whoami">أنا مونيرويو</string>
+    <string name="about_version">النسخة %1$s (%2$d)</string>
+
+    <string name="credits_text"><![CDATA[
+        <b>الاعتمادات</b>
+        <br/>
+        m2049r, baltsar777, anhdres, keejef,
+        rehrar, EarlOfEgo, ErCiccione et al.
+        <br/><br/>
+        <a href="https://monerujo.io/">monerujo.io</a>
+        ]]></string>
+
+    <string name="سياسة الخصوصية"><![CDATA[
```

Comment:
String name must be `privacy_policy` here. This shoud fix CircleCI build issue

Reply:
I just changed it. should build now.